integrations/categories.yaml

```diff
@@ -413,3 +401,27 @@
       most_popular: false
       priority: -1
       children: []
+- id: export
+  name: exporters
+  description: "Exporter Integrations"
+  most_popular: true
+  priority: 4
+  children:
+- id: notify
```

Comment:
@sashwathn does this need to be indented?

Reply:
I copied from the sections above but let me check.